docs/source/examples/performance_modeling.py

```diff
@@ -116,5 +117,9 @@ def setup(self):
 AC = Aircraft()
 MISSION = Mission(AC)
 M = Model(MISSION.takeoff_fuel, [MISSION, AC])
-SOL = M.solve(verbosity=0)
-print SOL.table()
+sol = M.solve(verbosity=0)
+
+vars_of_interest = set(AC.varkeys)
+vars_of_interest.update(MISSION.fs.aircraftp.unique_varkeys)
+vars_of_interest.add("D")
+print sol.summary(vars_of_interest)
```

Comment:
hmm, this seems overly complex and I'm not sure these are the only variables of interest. I think we should just print `sol.table()` for this model